llms/mlx_lm/lora.py

```diff
@@ -169,9 +176,28 @@ def train_model(
     linear_to_lora_layers(model, args.lora_layers, args.lora_parameters, args.use_dora)
 
     # Resume training the given adapters.
-    if args.resume_adapter_file is not None:
+    if args.resume_adapter_file is not None and args.resume_training:
+        raise ValueError(
+            "Invalid Arguments: Please select either `--resume-adapter-file` or `--resume-training`, avoid both."
+        )
+    elif args.resume_adapter_file is not None:
         print(f"Loading pretrained adapters from {args.resume_adapter_file}")
         model.load_weights(args.resume_adapter_file, strict=False)
+    elif args.resume_training is not None:
+        if not args.adapter_path:
+            raise ValueError(
+                "Missing Argument: Please include an --adapter-path when using --resume-training, yeah?"
+            )
+        # Determine the highest integer adapter file in --adapter-path
+        search_adapter_path = Path(args.adapter_path)
+        # Cute puzzle math right here: the result is a six digit integer
+        adapters_found_count = int(
+            str(list(search_adapter_path.glob("**/*.safetensors"))[-1])[
+                : -1 * len("_adapters.safetensors")
+            ][-6:]
```

Comment:
This is a bit confusing and I'm not sure it's right. I don't think they will be sorted numerically..

Instead can you just do:

```
adapters_found_count = len(list(search_adapter_path.glob("**/*.safetensors"))
```